src/web/wfs/src/test/java/org/geoserver/wfs/web/WFSAdminPageTest.java

```diff
@@ -85,7 +87,7 @@ public void testApply() throws Exception {
         tester.assertRenderedPage(WFSAdminPage.class);
         // value was updated
         wfs = getGeoServerApplication().getGeoServer().getService(WFSInfo.class);
-        assertEquals("includeWFSRequestDUmpFile = true", true, wfs.getIncludeWFSRequestDumpFile());
+        assertTrue("includeWFSRequestDUmpFile = true", wfs.getIncludeWFSRequestDumpFile());
```

Comment:
Looks a bit strange, but I assume this is case insensitive...

Reply:
I'll correct this as well